packages/cozy-device-helper/src/platform.js

```diff
@@ -24,4 +24,4 @@ export const isIOS = () =>
 // isMobile checks if the user is on a smartphone : native app or browser
 export const isMobile = () => isAndroid() || isIOS()
 
-export const isFlagshipApp = () => window.cozy?.isFlagshipApp
+export const isFlagshipApp = () => Boolean(window.cozy?.isFlagshipApp)
```

Comment:
let's add a test and/or jsdoc about it 🙏

Reply:
well there is already a test, it's still passing

Reply:
1) I think we should update the test to prevent regression (of reverting your commit change for example):

```
  it('should identify as a Flagship app webview', () => {
    window.cozy = undefined
    expect(isFlagshipApp()).toBe(false)
    window.cozy = {}
    expect(isFlagshipApp()).toBe(false)
    window.cozy = { isFlagshipApp: '' }
    expect(isFlagshipApp()).toBe(false)
    window.cozy = { isFlagshipApp: true }
    expect(isFlagshipApp()).toBe(true)
  })

```

Reply:
2. I prefer to test platform inside platform.spec.js rather than index.spec.js , it's more straightforward, to find matching test. Could you move the test please?

Reply:
updated